core/ui/EditTemplate/fields.tid

```diff
@@ -15,7 +15,9 @@ $:/config/EditTemplateFields/Visibility/$(currentField)$
 \end
 
 \define new-field-actions()
-<$action-sendmessage $message="tm-add-field" $name={{{ [<newFieldNameTiddler>get[text]] }}} $value={{{ [<newFieldValueTiddler>get[text]] }}}/>
+<$set name="newFieldValue" tiddler=<<newFieldValueTiddler>> field={{{ [<newFieldNameTiddler>get[text]] }}} >
+<$action-sendmessage $message="tm-add-field" $name={{{ [<newFieldNameTiddler>get[text]] }}} $value=<<newFieldValue>>/>
```

Comment:
Why do we use a `$set` widget for `newFieldValue` instead of a filtered transclusion in `$action-sendmessage`?

Reply:
Hello @saqimtiaz, I actually need your help here: The `field` attribute on this `$set` widget is already a filtered transclusion. I use the `$set` widget because I need to get the field, named by the `newFieldName`-Tiddler's `text` field, from the `newFieldValueTiddler`. I have no idea how I do this within a single filtered transclusion.

Reply:
If I understand correctly this should do the trick: 
`[<newFieldValueTiddler>get{newFieldNameTiddler}]`

otherwise:

`[<newFieldNameTiddler>get[text]] :map[<newFieldValueTiddler>get<currentTiddler>]`